### PR TITLE
Jj initcode

### PR DESCRIPTION
**Pull request by JorMaister**

### CODE REVIEW

File: cryocare/protocols/protocol_denoise.py [hunk 1]
```diff
@@ -49,40 +49,37 @@ def _defineParams(self, form):
...
+                      help='Tomogram reconstructed from the even frames of the tilt'
+                           'series movies.')
 
-        form.addParam('times', params.IntParam,
-                      default=10,
-                      label='Times', important=True,
-                      help='Times the message will be printed.')
+        form.addParam('oddTomo', params.PointerParam,
+                      pointerClass='Tomogram',
+                      label='Tomogram 1 (odd)',
```
Comment (pconesa) [inline cryocare/protocols/protocol_denoise.py:61]: Tomogram 2?

Comment (pconesa) [reply cryocare/protocols/protocol_denoise.py:61]: For denoising.....do we actually need the odd and even ones? or just a noisy tomogram.

File: cryocare/protocols/protocol_training.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        form.addParam('evenTomo', params.PointerParam,
+                      pointerClass='Tomogram',
+                      label='Tomogram 1 (even)',
+                      important=True,
+                      help='Tomogram reconstructed from the even frames of the tilt'
+                           'series movies.')
+
+        form.addParam('oddTomo', params.PointerParam,
+                      pointerClass='Tomogram',
+                      label='Tomogram 1 (odd)',
```
Comment (pconesa) [inline cryocare/protocols/protocol_training.py:60]: Tomogram 2?

File: cryocare/protocols/protocol_training.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        summary = []
+
+        if self.isFinished():
+            summary.append("This protocol has printed *%s* %i times." % (self.message, self.times))
+        return summary
+
+    def _methods(self):
+        methods = []
+
+        if self.isFinished():
```
Comment (pconesa) [inline cryocare/protocols/protocol_training.py:93]: This can go away

File: cryocare/protocols/protocol_denoise.py (hunk 1 above)
Comment (pconesa) [reply cryocare/protocols/protocol_denoise.py:61]: I can see we do need them.

Comment (azazellochg) [reply cryocare/protocols/protocol_denoise.py:61]: If it's even and odd halves, it must be tomo 1 (same tomogram) otherwise it will be a disaster.

Comment (pconesa) [reply cryocare/protocols/protocol_denoise.py:61]: Maybe just removing the number will do. Actually there is a chance that we can input a set of tomograms and avoid in the denoise the odd/even stuff, leaving it to just the training. Depends on the results

**Pull request merged**